ydb/core/tx/scheme_board/subscriber.cpp

```diff
@@ -1040,6 +1040,10 @@ class TSubscriber: public TMonitorableActor<TDerived> {
                 this->Send(proxy, new TEvents::TEvPoisonPill());
             }
         }
+        if (CurrentSyncRequest) {
+            this->Send(Owner, new NInternalEvents::TEvSyncResponse(Path, true), 0, CurrentSyncRequest);
+            CurrentSyncRequest = 0;
+        }
```

Comment:
```suggestion

        if (CurrentSyncRequest) {
            this->Send(Owner, new NInternalEvents::TEvSyncResponse(Path, true), 0, CurrentSyncRequest);
            CurrentSyncRequest = 0;
        }

```
Можно так, чтобы визуально отделить.